### ISSUE TITLE
Add player profile `beatmapGraveyard` event

### ISSUE THREAD

**Issue by Sebola3461** [ppy/osu-web#10134]
it currently displays these statuses:
![image](https://user-images.githubusercontent.com/62405013/233200281-30bf5c54-1159-43a9-a77f-fd151cef8976.png)
but there's not an event for graveyard beatmaps. It would be a cool addition (and simple too)

**Comment by peppy** [comment]
I see no reason not to add it. Should be a simple one.

**Comment by stilettocode** [comment]
Should I give this a try?

**Comment by peppy** [comment]
Go for it.

**Comment by Jomo178** [comment]
Hey @stilettocode, are you still planning to work on this? If not I'd be happy to take it on. Please lmk thanks!!

**Comment by keilogic** [comment]
Hi, I’d be happy to take this on if it’s still available.

Before I start, I wanted to ask whether this kind of small `good first issue` is eligible for a bounty request as mentioned in the contribution docs. PayPal works for me if a bounty is approved.

If it’s available, my plan is to trace the existing beatmapset status event handling, add the missing `beatmapsetGraveyard` event path, include the locale/display text, and add/update the focused tests or fixtures that cover profile event rendering.

**Comment by peppy** [comment]
no, we don't do bounties. and it sounds like you may be about to use an LLM, so please read our README first to avoid getting banned.
